en/components/overlay_main.md

```diff
@@ -30,7 +28,7 @@ export class MyOverlayComponent {
 ...
 ```
 
-### Displaying overlay content
+## Displaying overlay content
```

Comment:
This should be an H3 headline, not H2. The same goes for every other change made below.